sass/_clearfix.scss

```diff
@@ -2,25 +2,14 @@
 // ========
 
 
-// Micro Clearfix
-// --------------
+// Cleafix
+// -------
 /// A small, unobtrusive clearfix for float containers.
 /// @group clearfix
-@mixin micro-clearfix {
-  &:after {
+@mixin clearfix {
```

Comment:
Well, we've always had the `clearfix` alias, so that's not a new problem. Doesn't worry me much, since most clearfixes are basically the same, this is a real common one, and you can override it by import-order if you really want to.